src/app.tsx

```diff
@@ -100,8 +162,65 @@ function App() {
       <Button variant="dark" onClick={dummy}>
         Hello
       </Button>
+
+      <div className="flashcards-container">
+        <Flashcard card={readonlyCard} variant={'readonly'} />
+        <Reorder.Group axis="y" values={cards} onReorder={setCards}>
+          {getFlashcards()}
+        </Reorder.Group>
+      </div>
     </div>
   );
+
+  function getFlashcards() {
+    return cards.map((card, id) => {
+      const index = id + 1;
+      const value = (
+        <Flashcard
+          key={index}
+          card={card}
+          index={index}
+          variant={activeCard === card.id ? 'active' : 'inactive'}
+          onFocus={() => setActiveCard(card.id)}
+          onCardChange={(c) => handleCardChange(c, id)}
+          onDownClick={() => handleDownClick(id)}
+          onUpClick={() => handleUpClick(id)}
+          onRemoveClick={() => handleRemoveClick(id)}
+        />
+      );
+      return (
+        <Reorder.Item key={card.id} value={card}>
+          {value}
+        </Reorder.Item>
+      );
+    });
+
+    function handleRemoveClick(id: number) {
+      const newCards = [...cards];
+      newCards.splice(id, 1);
+      setCards(newCards);
+    }
+
+    function handleUpClick(id: number) {
+      // Todo handle up click when card is first
+      const newCards = [...cards];
+      newCards[id] = newCards.splice(id - 1, 1, cards[id])[0];
+      setCards(newCards);
+    }
+
+    function handleDownClick(id: number) {
+      //Todo handle down click when card is last
```

Comment:
Is this something you want to fix before demo/merging this? 

If not, can you track this either by opening an issue or adding it to our trello board?

Reply:
This was just a makeshift parent component to render the flashcards as a group. I am working on the real parent right now and will have it fixed